models/Order.js

```diff
@@ -50,8 +72,14 @@ const orderSchema = new mongoose.Schema({
       quantity: Number,
     },
   ],
-  totalPrice: [CurrencySet],
-  paymentMethod: String,
+  totalItemsPrice: [CurrencySet],
+  totalPriceToPay: [CurrencySet],
+  paymentMethod: {
+    type: String,
+    required: true,
+    enum: ['card', 'cash'],
```

Comment:
Please, make it uppercased with underscores, as for `status`, let's follow the same formatting for all enum values

Reply:
Done